src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewAccessibleObject.cs

```diff
@@ -0,0 +1,240 @@
+
+using System.Collections.Generic;
+using System.Drawing;
+using System.Runtime.InteropServices;
+using Accessibility;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class ListView
+    {
+        [ComVisible(true)]
+        internal class ListViewAccessibleObject : ControlAccessibleObject
+        {
+            private readonly IAccessible _systemIAccessible;
+
+            internal ListViewAccessibleObject(ListView owner) : base(owner)
+            {
+                _systemIAccessible = GetSystemIAccessibleInternal();
+            }
+
+            internal override UiaCore.IRawElementProviderFragmentRoot FragmentRoot => null;
+
+            private ListView OwningListView => Owner as ListView;
+
+            internal override bool IsPatternSupported(UiaCore.UIA patternId)
+            {
+                if (patternId == UiaCore.UIA.SelectionPatternId ||
+                    patternId == UiaCore.UIA.MultipleViewPatternId ||
+                    patternId == UiaCore.UIA.LegacyIAccessiblePatternId ||
+                    (patternId == UiaCore.UIA.GridPatternId && OwningListView.View == View.Details) ||
+                    (patternId == UiaCore.UIA.TablePatternId && OwningListView.View == View.Details))
+                {
+                    return true;
+                }
+
+                return base.IsPatternSupported(patternId);
+            }
+
+            internal override UiaCore.IRawElementProviderFragment FragmentNavigate(UiaCore.NavigateDirection direction)
+            {
+                int childCount = OwningListView.Items.Count;
+
+                if (childCount == 0)
+                {
+                    return null;
+                }
+
+                return direction switch
+                {
+                    UiaCore.NavigateDirection.FirstChild => GetChild(0),
+                    UiaCore.NavigateDirection.LastChild => GetChild(childCount - 1),
+                    _ => base.FragmentNavigate(direction)
+                };
+            }
+
+            internal override bool CanSelectMultiple => true;
+
+            internal override object GetPropertyValue(UiaCore.UIA propertyID)
+                => propertyID switch
+                {
+                    UiaCore.UIA.NamePropertyId
+                        => Name,
+                    UiaCore.UIA.AutomationIdPropertyId
+                        => OwningListView.IsHandleCreated ? OwningListView.Name : string.Empty,
+                    UiaCore.UIA.RuntimeIdPropertyId
+                        => RuntimeId,
+                    UiaCore.UIA.ControlTypePropertyId
+                        => UiaCore.UIA.ListControlTypeId,
+                    UiaCore.UIA.IsMultipleViewPatternAvailablePropertyId
+                        => IsPatternSupported(UiaCore.UIA.MultipleViewPatternId),
+                    UiaCore.UIA.ItemStatusPropertyId
+                        => GetItemStatus(),
+                    _ => base.GetPropertyValue(propertyID)
```

Comment:
[readability] Keep on the same line
```suggestion
                    UiaCore.UIA.NamePropertyId => Name,
                    UiaCore.UIA.AutomationIdPropertyId => OwningListView.IsHandleCreated ? OwningListView.Name : string.Empty,
                    UiaCore.UIA.RuntimeIdPropertyId => RuntimeId,
                    UiaCore.UIA.ControlTypePropertyId => UiaCore.UIA.ListControlTypeId,
                    UiaCore.UIA.IsMultipleViewPatternAvailablePropertyId => IsPatternSupported(UiaCore.UIA.MultipleViewPatternId),
                    UiaCore.UIA.ItemStatusPropertyId => GetItemStatus(),
                    _ => base.GetPropertyValue(propertyID)
```

Reply:
Fixed. Thanks.